src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -879,6 +886,7 @@
          boxing.  If this node has errors, then the conversion may not be present.-->
     <Field Name="Expression" Type="BoundExpression"/>
     <Field Name="DeconstructionOpt" Type="BoundForEachDeconstructStep" Null="allow"/>
+    <Field Name="AwaitOpt" Type="AwaitableInfo" Null="allow"/>
```

Comment:
Is there any info here that we want to expose as part of IOperation? @dotnet/analyzer-ioperation @AlekseyTs. At the least, I'd think we want to have a bool field on IForEachLoopOperation that says whether this is an async foreach.

Reply:
I have a [work item](https://github.com/dotnet/roslyn/issues/24037) to handle IOperation in a later PR.